### DISCUSSION TITLE
Battle goals

### DISCUSSION THREAD

**Discussion by ainaerco** [Lurkars/gloomhavensecretariat#109] [Ideas]
I have made json file with battle goals for Frosthaven. Will it be useful to add it to the repo?

**Comment by Lurkars** [comment]
By now I have no plans to implement battlegoals. But this may still come as feature in future, so just leave a link here for reference! Will be useful then!

**Reply by ainaerco** [reply]
Will try to put it in some accessible place.
I noticed myself missing out to select battle goals and finding out only after couple of turns after the scenario begun. Will it be feasible to show a reminder somewhere if there are no plans to add selecting goals feature soon?

**Reply by pikdonker** [reply]
Hey @ainaerco, have you uploaded that battle goals json somewhere? I have some use for it, would save me from creating it myself.. Thanks!

**Reply by korvegan** [reply]
@Lurkars regarding the battle goals, are thse implemented ? I would like to contribute and translate them to German.

**Reply by Lurkars** [reply]
Yes in the meanwhile I implemented battle goals! If you want to provide translaten, take a look at english files
- FH https://github.com/Lurkars/gloomhavensecretariat/blob/3e04dd25870d137c9f8a196e322da508790f5575/data/fh/label/en.json#L168
- GH https://github.com/Lurkars/gloomhavensecretariat/blob/3e04dd25870d137c9f8a196e322da508790f5575/data/gh/label/en.json#L173
- JOTL https://github.com/Lurkars/gloomhavensecretariat/blob/3e04dd25870d137c9f8a196e322da508790f5575/data/jotl/label/en.json#L65
- https://github.com/Lurkars/gloomhavensecretariat/blob/3e04dd25870d137c9f8a196e322da508790f5575/data/sc/label/en.json#L5

Just copy those over to the correpsonding `de.json` files and change translation.

**Reply by korvegan** [reply]
Okay, will use that topic for status update, so battle
Goals are translated. Now I will translate the rest of the file, monster abilities are left and then I will take care of the character abilities.

**Comment by Lurkars** [comment]
I just forgot to mention it here, so support for Battle Goals was added a while ago in [v0.68.0](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.68.0) and this discussion can be closed.
